docs/_entities/embed.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Embed
+---
+
+This entity represents the embed information about an URL that should be
+embedded, it is nested in a [StatusMessage][status_message].
+
```

Comment:
I'm a bit unsure about that, I think that sentence points a bit more to the oembed direction (obviously, because you based it from oembed.com), because I don't think opengraph matches "representation of a third party resource", because opengraph is more a preview than a representation. But I also don't know how to write it better (to describe both). So until we know something better, it's probably better than nothing, so I added it anyway. :) If you know something better, let me know :)

(Also thanks for the review :cookie:, everything else is fixed too :) )

Reply:
Maybe then just add the `preview` word there: "... to keep an embedded representation **or a preview** of a third party resource ..." ?

Reply:
Sometimes it can be that easy :D added :+1: